amical/oifits.py

```diff
@@ -580,8 +580,8 @@ def save(
     hdu.header["ORIGIN"] = origin or hdr.get("ORIGIN", "Sydney University")
     hdu.header["CONTENT"] = "OIFITS2"
     hdu.header["DATE-OBS"] = hdr.get("date-obs", "")
-    hdu.header["TELESCOP"] = hdr.get("TELESCOP", "")
-    hdu.header["INSTRUME"] = hdr.get("INSTRUME", "")
+    hdu.header["TELESCOP"] = hdr.get("telescop", "")
```

Comment:
this change doesn't seem fully backward compatible. Can you explain ?
was the previous implementation (all caps) wrong, or is this about adding compatibility with a newer format ? In this second case, I don't think it warrants breaking compatibility with the older version, and we should be able to work with both.

Reply:
If I remember correctly it was wrong, because of this change: 

https://github.com/SydneyAstrophotonicInstrumentationLab/AMICAL/blob/d4c6943aee310d535429b7ff458c98fa0ede39f1/amical/mf_pipeline/bispect.py#L1050-L1061

This change is actually from #136, which addresses this specifically. I will add a comment below to say that this PR depends on #136.